app/models/subscription.rb

```diff
@@ -12,10 +12,6 @@ class Subscription < ApplicationRecord
   validates :email_address, format: { with: URI::MailTo::EMAIL_REGEXP, allow_nil: true }, presence: false
   validate :email_address_or_callback_url_required
 
-  def kept?
-    !discarded?
-  end
-
```

Comment:
This is superfluous and is provided by the Discard gem: https://github.com/jhawthorn/discard/blob/master/lib/discard/model.rb#L113

Reply:
good spot 👍